flexible-classroom/develop/record-a-class.mdx

```diff
@@ -25,7 +25,14 @@ If you want to implement recording on your own, you can refer to the following d
 
 ![implement recording on your own](https://web-cdn.agora.io/docs-files/1638259107675)
 
+<PlatformWrapper notAllowed="android">
+<PlatformWrapper notAllowed="ios">
 When you deploy the web page to be recorded into your CDN, you can use the template HTML file `templates/record_page_prod.html` provided by Agora in the [Flexible-Classroom-Desktop](https://github.com/AgoraIO-Community/flexible-classroom-desktop) GitHub repository.
```

Comment:
1. I have built this and it doesn't seem to work - the repo is still the same for all 4 platforms. 
2. You need to not only exclude Android and iOS, but also provide a separate entry for each of them, specifying correct repos.